pyfr/shapes.py

```diff
@@ -129,10 +129,6 @@ def m6(self):
     def m7(self):
         return self.ubasis.nodal_basis_at(self.qpts)
 
-    @lazyprop
```

Comment:
Thoughts on renaming 9 -> 8 etc.

Reply:
m5 is also missing. Should we start with m6 -> m5 or is there a reason for the missing 5?

Reply:
m5 a 2x2 or 3x3 diagonal block of m1 matrices.  For efficiency we never form it directly but it is still 'there' in the formal sense.